server/meca/routes.js

```diff
@@ -37,19 +37,33 @@ module.exports = app => {
 
     const updateStatus = () => Manuscript.updateStatus(manuscriptId, status)
 
-    const sendEmail = () => mailer.send({
-      to: config.get('meca.email.recipient'),
-      from: config.get('meca.email.sender'),
-      subject: `${config.get('meca.email.subjectPrefix')}MECA import failed`,
-      text: `
+    const auditlog = () =>
+      new AuditLog({
+        action: 'UPDATED',
+        objectId: manuscriptId,
+        objectType: 'ejp.id',
+        value: body.id,
+      }).save()
```

Comment:
Is it possible to add a unit test to ensure `.save()` is called on and `AuditLog` object?

Reply:
It's easier to do the same as https://github.com/elifesciences/elife-xpub/blob/develop/server/xpub-server/entities/manuscript/resolvers/uploadSupportingFile.test.js for now. I don't think we've got a test example where we mock out `.save()`